Python/dynamic/SS.py

```diff
@@ -103,19 +63,16 @@ def create_steady_state_parameters(a_tax_income, b_tax_income, c_tax_income,
     #for key in variables:
     #    globals()[key] = variables[key]
     if get_baseline is False:
+        # If this is a tax experiment, also import the changed tax variables
         variables = pickle.load(open("OUTPUT/Saved_moments/params_changed.pkl", "r"))
         for key in variables:
             globals()[key] = variables[key]
 
     # Make a vector of all one dimensional parameters, to be used in the following functions
-    income_tax_params = [a_tax_income, b_tax_income, c_tax_income,
-                         d_tax_income]
+    income_tax_params = [a_tax_income, b_tax_income, c_tax_income, d_tax_income]
     wealth_tax_params = [h_wealth, p_wealth, m_wealth]
     ellipse_params = [b_ellipse, upsilon]
-    parameters = ([J, S, T, beta, sigma, alpha, Z, delta, ltilde, nu, g_y,
-                  tau_payroll, retire, mean_income_data] + income_tax_params
-                  + wealth_tax_params + ellipse_params)
-
+    parameters = [J, S, T, beta, sigma, alpha, Z, delta, ltilde, nu, g_y, g_n_ss, tau_payroll, retire, mean_income_data] + income_tax_params + wealth_tax_params + ellipse_params
     iterative_params = [maxiter, mindist_SS]
```

Comment:
I didn't even notice it did that.  I changed what was in that list, and so I just copy and pasted over it.  It probably did that in a few places, I can go through and fix that.